content/doc/book/blueocean/creating-pipelines.adoc

```diff
@@ -64,33 +63,37 @@ repository in the *Repository URL* field.
 image:blueocean/creating-pipelines/connect-to-a-git-repository.png['Connect to a
 Git repository',width=70%]
 
-You now need to specify a <<local-repository,local>> or a
-<<remote-repository,remote>> repository from which to build your Pipeline
+You now need to specify a link:#local-repository[local] or a
+link:#remote-repository[remote] repository from which to build your Pipeline
 project.
 
 
 ==== Local repository
 
 If your URL is a local directory path (e.g. beginning with a forward slash `/`
-such as `/home/cloned-git-repositories/my-git-repository.git`), you can proceed
-to click the *Create Pipeline* button.
+such as `/home/cloned-git-repos/my-git-repo.git`), you can proceed to click the
```

Comment:
I thought it might be worthwhile shortening these. "Repositories" is a long word.